lib/Echidna/Transaction.hs

```diff
@@ -250,6 +250,13 @@ shrinkTx tx'@(Tx c _ _ _ gp (C _ v) (C _ t, C _ b)) = let
     ]
   in join (uniform possibilities) <*> pure tx'
 
+mutateTx :: (MonadRandom m, Has GenDict x, MonadState x m, MonadThrow m) => Tx -> m Tx
```

Comment:
Is `Has GenDict x, MonadState x m, MonadThrow m` needed here?

Reply:
Fixed.